metrics/metrics.go

```diff
@@ -87,7 +89,24 @@ func InitPrometheus(prometheusHost, prometheusPort, prometheusEntry string) {
 	handler := promhttp.HandlerFor(promReg, promhttp.HandlerOpts{EnableOpenMetrics: true})
 	http.Handle("/"+prometheusEntry, handler)
 	go func() {
-		glog.Infof("Starting Prometheus on %v:%v, entry point is /%v", prometheusHost, prometheusPort, prometheusEntry)
+
+		if strings.HasPrefix(prometheusHost, "unix:") {
+			socketPath := prometheusHost[5:]
+			glog.Infof("Starting Prometheus on Unix socket %v, entry point is /%v", socketPath, prometheusEntry)
+			os.Remove(socketPath) // allow failure
```

Comment:
_ := os.Remove(socketPath)

Reply:
https://github.com/shizunge/endlessh-go/pull/128/commits/fdc4829b7c805a25aa7088aa609e606a75c49ccb

Reply:
actually i think it needs to be `_ = os.Remove(socketPath)` (no colon)